corehq/apps/app_manager/tests/test_filters.py

```diff
@@ -327,7 +327,7 @@ def test_ancestor_location_type_filter_with_no_location_found(self):
         self.assertEqual(ancestor_state_value, ui_filter.default_value())
 
 
-class NumericFilterTests(TestCase):
+class NumericFilterTests(SimpleTestCase):
```

Comment:
Unrelated to the rest of this work - was poking around tests to try to figure out how all this stuff works